src/Shark_Cheats.md

```diff
@@ -28,8 +28,8 @@ Check the [Game Genie manual](http://www.digitpress.com/library/manuals/gameboy/
 Game Shark codes consist of eight-digit hex numbers, formatted as
 ABCDEFGH, the meaning of the separate digits is:
 
-` AB    External RAM bank number`
-` CD    New Data`
+` AB    External RAM bank number` \
```

Comment:
Imo, this should be a table instead of pre-formatted text, for accessibility.

Reply:
I agree, a table would be better here